omop_file_validator.py

```diff
@@ -898,4 +946,22 @@ def evaluate_submission(d):
 
 
 if __name__ == '__main__':
-    evaluate_submission(settings.csv_dir)
+    parser = argparse.ArgumentParser(
+        description=
+        "Evaluate OMOP files for formatting issues before AoU submission.")
+
+    parser.add_argument('-r',
+                        '--restrict',
+                        action='store_true',
+                        help="Flag to restrict rows by argument {--n}")
+    parser.add_argument('-n',
+                        '--n_rows',
+                        type=int,
+                        default=1000,
+                        help="Number of rows to validate per file.")
```

Comment:
I would suggest using only one arg for this
```suggestion
    parser.add_argument('-r',
                        '--restrict',
                        action='store',
                        required=False,
                        help="Number of rows to restrict for validation per file. e.g. --restrict 1000 for only validating the first 1000 lines")
```